package.json

```diff
@@ -1,22 +1,23 @@
 {
 	"name": "@pressbooks/pressbooks-cas-sso",
 	"description": "CAS Single Sign-On integration for Pressbooks.",
-  "scripts": {
-    "install-build-tools": "npm install --no-save pressbooks-build-tools",
-    "watch": "npm run production -- --watch",
-    "prod": "npm run production",
-    "build:production": "npm run production",
-    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "test": "npm run lint",
-    "lint": "npm run -s lint:scripts && npm run -s lint:styles",
-    "lint:scripts": "cross-env NODE_ENV=development node_modules/eslint/bin/eslint.js \"assets/src/scripts/*.js\"",
-    "lint:styles": "cross-env NODE_ENV=development node_modules/stylelint/bin/stylelint.js \"assets/src/styles/**/*.scss\" --syntax scss"
-  },
+	"scripts": {
+		"watch": "mix watch",
+		"prod": "npm run production",
```

Comment:
I feel we don't need to production commands I think we can get rid of "prod" one

Reply:
Agree. I removed the 'prod' command.